imap/jmap_ical.c

```diff
@@ -3356,11 +3376,18 @@ participants_to_ical(icalcomponent *comp,
             int i;
             for (i = 0; i < ptrarray_size(complist); i++) {
                 icalcomponent *tmp = ptrarray_nth(complist, i);
-                if ((icalcomponent_get_first_property(tmp, ICAL_ORGANIZER_PROPERTY) == NULL) !=
-                     (icalcomponent_get_first_property(tmp, ICAL_ATTENDEE_PROPERTY) == NULL)) {
-                    allow_organizer_attendee_only = 1;
-                    break;
+                icalproperty *orga = icalcomponent_get_first_property(tmp, ICAL_ORGANIZER_PROPERTY);
+                icalproperty *attd = icalcomponent_get_first_property(tmp, ICAL_ATTENDEE_PROPERTY);
+                allow_organizer_attendee_only = (orga == NULL) != (attd == NULL);
+                if (!allow_organizer_attendee_only) {
+                    char *orgauri = normalized_uri(icalproperty_get_value_as_string(orga));
+                    char *attduri = normalized_uri(icalproperty_get_value_as_string(attd));
+                    allow_organizer_attendee_only = (orgauri == NULL) != (attduri == NULL);
+                    free(orgauri);
+                    free(attduri);
                 }
+                if (allow_organizer_attendee_only)
```

Comment:
Couldn't this be 'else if'

Reply:
Not sure I understand: Line 3395 updates `allow_organizer_attendee_only` and we want to break if either the property isn't set at all, or has a null value.

Reply:
I'd probably recommend an extra blank link between the blocks in that case, so it's clear it's not a continuation case :)  First item for our code review review maybe!

* if it's an `else if` there should be no blank line, if it's a new `if` then there should be a blank line before the new block.

Reply:
@rsto My mistake, I missed the fact that allow_organizer_attendee_only was being set in the if {} block.  A blank line would definitely help